lib/fast_jsonapi/object_serializer.rb

```diff
@@ -104,6 +120,7 @@ def inherited(subclass)
         subclass.race_condition_ttl = race_condition_ttl
         subclass.data_links = data_links
         subclass.cached = cached
+        subclass.fieldset = fieldset.dup if fieldset.present?
```

Comment:
fieldsets needs to be passed in each time `serializable_hash` is invoked. I don't think it makes sense to inherit this when subclassing.

Reply:
That's another remnant I failed to cleanup. Should be gone after the next commit.